programmers/제일-작은-수-제거하기-sort오류.js

```diff
@@ -0,0 +1,6 @@
+function solution(arr) {
+    arr.sort((a, b) => (b-a))
+    if (arr.length < 2) return [-1]
```

Comment:
arr.length를 변수로 선언해주세요~
 2보다 작다 말고 === 1 이 조건에 명확하네요 
```suggestion
    if (arr.length === 1) return [-1]
```